app/styles/layouts.scss

```diff
@@ -62,37 +62,123 @@
   min-height: 520px;
 }
 
-.l-welcome-header {
+.l-welcome_container {
+  width: 416px;
+  height: 488px;
+  border-radius: 4px;
+  background-color: #fff;
+  box-shadow: 0 1px 3px 0 rgba(0, 0, 0, .1), 0 1px 2px 0 rgba(0, 0, 0, .2);
   position: relative;
-  width: 100%;
-  height: 40vh;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.l-welcome_container.reset_pas {
+  height: 318px;
```

Comment:
We decided to use 4 spaces (https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web#general)